web3/contract.ts

```diff
@@ -21,8 +21,11 @@ export const getQuestChainContract = (
       signer,
     ) as contracts.V1.QuestChain;
   }
-  return contracts.V2.QuestChain__factory.connect(
-    address,
-    signer,
-  ) as contracts.V2.QuestChain;
+  if (version === '2') {
```

Comment:
hey lets return V2 contract by default, instead of throwing an error. This was the older logic, but it broke if we potentially push subgraph for v3 contracts before we publish v3 UI.

Reply:
fixed in [c0104f6](https://github.com/quest-chains/dapp/pull/143/commits/c0104f6ae2a04232f0630666ca4411862c4cb61f)